daemon/hostdaemon.go

```diff
@@ -51,21 +56,37 @@ func NewHostDaemon(vsp plugin.VendorPlugin) *HostDaemon {
 	}
 }
 
+func (d *HostDaemon) ensureConnected() {
+	if d.conn != nil {
```

Comment:
When you `Dial` using `grpc`, the connection might not be up. For that reason, having an additional check here might not work. If there is no connection, it will error out later.